include/opendht/callbacks.h

```diff
@@ -56,6 +56,8 @@ struct OPENDHT_PUBLIC NodeStats {
      * Build a json object from a NodeStats
      */
     Json::Value toJson() const;
+    NodeStats() {};
```

Comment:
The bug comes from here. If you add a constructor, you must give default values (0) otherwise fields are not default-initialized